src/components/Editor/EditorOutline.vue

```diff
@@ -61,10 +61,14 @@ export default {
 	// 204px = 50px nc header + 60px collectives titlebar + 44px menubar + 50px bottom margin
 	max-height: calc(100% - 204px);
 
+	border-radius: var(--border-radius-large);
+	box-shadow: 0 1px 10px var(--color-box-shadow);
+	background-color: var(--color-main-background);
```

Comment:
Can we drop this change and rather discuss it separately?

Having the outline floating without borders was part of the design requirements or feedback when we introduced it, so I'd not change that as part of a backport at least.

Reply:
Yeah, sorry for pushing two unrelated fixes into one PR. I changed this PR to only contain the menubar fix now.